en-US/Revision_History.xml

```diff
@@ -8,7 +8,36 @@
 	 <simpara>
 		<revhistory>
 			<revision>
-				<revnumber>3-0</revnumber>
+				<revnumber>5-1</revnumber>
+				 <date>January 2022</date>
+				 <author>
+					<firstname>Julian</firstname>
+					 <surname>Cable</surname>
+					 <email>jcable@redhat.com</email>
+				</author>
+				 <revdescription>
+					<simplelist>
+						<member>Minor update to address some reported issues:</member>
+						<member>Accessing IBM Style online.</member>
+						<member>Updated guidance for long commands.</member>
+						<member>Updated guidance for referring to object names.</member>
+						<member>Updated guidance for non-breaking spaces.</member>
+						<member>All punctuation, including periods and commas, is now placed outside quotation marks unless the punctuation is part of the quotation.</member>
+						<member>The gerund verb form is no longer required in section titles.</member>
+						<member>Pronouncing file or directory names that begin with special characters.</member>
+						<member>Avoid "the product allows the user to do xyz".</member>
+						<member>Avoid consecutive headings.</member>
+						<member>Guidance to refer to UI elements and to other publication titles.</member>
+						<member>Clarification that diagram labels use sentence case.</member>
+						<member>New or updated usage entries: backend, IaC, number sign, on premise, prebaked, unset.</member>
```

Comment:
"On-premise" needs to be hyphenated.

Reply:
I've updated the usage entry to show both unhyphenated and hyphenated forms.